bitnami/mariadb/templates/secrets.yaml

```diff
@@ -9,6 +9,12 @@ metadata:
     chart: "{{ template "mariadb.chart" . }}"
     release: {{ .Release.Name | quote }}
     heritage: {{ .Release.Service | quote }}
+  {{- if .Values.secret.annotations }}
+  annotations:
+    {{- range $key, $value := .Values.secret.annotations }}
+    {{ $key }}: {{ $value }}
+    {{- end }}
```

Comment:
I would use `mysql.tplValues`

```
 annotations: {{- include "mysql.tplValue" (dict "value" .Values.secret.annotations "context" $) | nindent 4 }}
```

Reply:
`mariadb.tplValues` is not available at [templates/_helpers.tpl](https://github.com/bitnami/charts/blob/master/bitnami/mariadb/templates/_helpers.tpl).